app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<header>
+ <%= link_to "Back", root_path, class: 'back_btn'%>
+
+ <h1>Transactions</h1>
+
+  <% if user_signed_in? %>
+    <%= button_to "Logout", destroy_user_session_path, method: :delete, data: {turbo: false}, class: "btn bg-danger"  %>
+  <% end %>
+
+</header>
+
+<%= render partial: "categories/category", locals: {category: @category} %>
+
+<div id="transactions">
+  <% @transactions.each do |transaction| %>
+    <%= render transaction %>
+  <% end %>
+</div>
+
+<footer>
+<%= link_to "Add Transaction", new_category_transaction_path %>
+</footer>
```

Comment:
- [ ] Please kindly ensure that the transactions and categories page matches the original design guidelines. See image below for reference

![image](https://user-images.githubusercontent.com/43995830/189247392-640a7fe1-c7d6-4501-99b2-15ed0f59b17f.png)